assets/sass/elements/_forms.scss

```diff
@@ -167,12 +158,45 @@ textarea.form-control {
 }
 
 textarea.form-control {
+  background-image: none;
+  // Remove margin under textarea in Chrome and FF
+  display: block;
   // Disable opacity and background image for Firefox
   opacity: 1;
-  background-image: none;
 }
-// scss-lint:enable QualifyingElement
 
+select.form-control {
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  appearance: none;
```

Comment:
Just in case anyone else stumbles over this... I initially assumed the `appearance` property doesn't exist. But it used to be in the spec, then it was removed and now it's in the next draft again. So it might be worth keeping it. See https://developer.mozilla.org/en-US/docs/Web/CSS/-moz-appearance